src/stylesheets/_forms.scss

```diff
@@ -236,7 +236,10 @@
         @extend %input-style;
         vertical-align: top;
         height: auto;
+        min-width: 15rem;
+        max-width: 100%;
         min-height: 9.2rem;
+        max-height: 30rem;
```

Comment:
Lets change this to 100vh instead of 30rem. If it needs to be less in a specific implementation, then it can be done in the css of that application.